From 8696f304f1ef598301029baffdf4e8abdcb90fe3 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Wed, 30 Aug 2017 17:23:26 +0000
Subject: [PATCH 18/18] gn_generator: fix debug build

Remove a duplicate CFLALGS value that contains spaces and
which will cause issues when interpreted by gn.
---
 mkspecs/features/gn_generator.prf | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/mkspecs/features/gn_generator.prf b/mkspecs/features/gn_generator.prf
index b2b7491..24561b8 100644
--- a/mkspecs/features/gn_generator.prf
+++ b/mkspecs/features/gn_generator.prf
@@ -19,6 +19,11 @@ defineReplace(filter_flag_values) {
         # the architecture itself.
         return("")
     }
+    contains(value_to_check, ".*-g -feliminate-unused-debug-types.*") {
+        # in debug mode, eliminate this spurious value, which contains spaces and will not be interpreted
+        # correctly by gn.
+        return("")
+    }
     return($$value_to_check)
 }
 
-- 
2.7.4

